system/include/webgpu/webgpu.h

```diff
@@ -1036,11 +1132,11 @@ typedef void (*WGPUProcShaderModuleReference)(WGPUShaderModule shaderModule);
 typedef void (*WGPUProcShaderModuleRelease)(WGPUShaderModule shaderModule);
 
 // Procs of Surface
-typedef void (*WGPUProcSurfaceGetPreferredFormat)(WGPUSurface surface, WGPUAdapter adapter, WGPUSurfaceGetPreferredFormatCallback callback, void * userdata);
```

Comment:
Keep

Reply:
Ok, although just to be sure, it's not implemented yet in `library_webgpu.js`
And also `WGPUAdapter` was removed in the process of updating the headers, so for this function to be valid I had to revert the removal of WGPUAdapter too

Reply:
Yeah, we'll need to implement it eventually. Right now it's not important because implementations (at least chrome) haven't yet even tried to return anything other than a constant `"bgra8unorm"` for it.